### PR TITLE
{}[intel/2016a] FLANN v1.8.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by wpoely86** [comment]
:+1:

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/ea3520fef63b69224097fd1a04b20a15 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2417.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/51b4698f74c13837e95314c72c8121b4 for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

File: easybuild/easyconfigs/f/FLANN/FLANN-1.8.4-intel-2016a-Python-2.7.11.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+modextrapaths = {'PYTHONPATH': ['share/flann/python']}
+
+sanity_check_paths = {
+    'files': ['lib/libflann_cpp_s.a', 'lib/libflann_s.a',
+              'lib/libflann_cpp.%s' % SHLIB_EXT, 'lib/libflann.%s' % SHLIB_EXT],
+    'dirs': ['include/flann', 'lib/pkgconfig', 'share/doc/flann', 'share/flann/python'],
+}
+sanity_check_commands = [('python', '-c "import pyflann"')]
+
+moduleclass = ''
```
Comment (boegel) [inline easybuild/easyconfigs/f/FLANN/FLANN-1.8.4-intel-2016a-Python-2.7.11.eb]: @wpoely86 tssk.... ;)

**Pull request merged**